automatedtests/src/androidTest/java/com/mux/stats/sdk/muxstats/automatedtests/ui/SimplePlayerBaseActivity.java

```diff
@@ -219,8 +219,7 @@ public boolean waitForPlaybackToStart(long timeoutInMs) {
   }
 
   public void waitForPlaybackToStartBuffering() {
-    if (player.getPlaybackState() == Player.STATE_READY &&
-        player.getPlayWhenReady()) {
+    if (!muxStats.isPaused()) {
```

Comment:
We can not access Player from different Thread it must be the UI thread, this is a fix for the issue.

Reply:
@tomkordic is this the main change needed to support `r2.13.1`? If there were additional changes, would you please call them out.

Reply:
This is only automated test related, I did comment here: https://github.com/muxinc/mux-stats-sdk-exoplayer/pull/98#discussion_r596475930 this is the only relevant change to the MuxStats, basically the changes in the API are minimal, most of this version is just copy of version 2.12.1